dotnet/csharp.bzl

```diff
@@ -447,6 +447,7 @@ def _nuget_package_impl(repository_ctx,
     nuget,
     "install",
     "-Version", repository_ctx.attr.version,
+    #"-Verbosity", "detailed", # enable for debugging
```

Comment:
Can you move this to be a regular comment at the top. I prefer to leave out commented code.

Alternatively, you can put in an issue to add add this to the command when a debugging flag is turned on.